lib/cisco_node_utils/cmd_ref/evpn_stormcontrol.yaml

```diff
@@ -0,0 +1,18 @@
+# evpn_stormcontrol.yaml
+---
+_exclude: [ios_xr, N3k, N5k, N6k, N7k, N9k-F]
+
+_template:
+  get_command: "show running-config"
```

Comment:
Same comment as above

Reply:
Modified this to 'show running-config | section storm-control'